mesonbuild/mdist.py

```diff
@@ -32,9 +32,10 @@
     from ._typing import ImmutableListProtocol
     from .mesonlib import ExecutableSerialisation
 
-archive_choices = ['gztar', 'xztar', 'zip']
+archive_choices = ['bztar', 'gztar', 'xztar', 'zip']
```

Comment:
nitpick: This list is only used in `determine_archives_to_generate` function, and could easily be replaced by using the keys of `archive_extension`. I would rather remove it...

Reply:
True. If someone else comments the same, I will remove it.

Reply:
I think it's fine as is.